.github/workflows/build-docs.yml

```diff
@@ -17,6 +17,7 @@
 #    - ``[skip doc]`` skip build of the documentation.
 #    - ``[example] name_of_example.py`` trigger build of some specific examples.
 #    - ``[force download]`` do not rely of cached datasets and force to redownload them.
+#    - ``[reports]`` build estimators reports that are embedded in the documentation.
```

Comment:
@hndgzkn 
it should now be possible to trigger the build of the reports embedded in the doc in CI by commit message (without needing to build the entire doc)